### PR TITLE
Remove jQuery when possible

### LINKED ISSUE #104: [jQuery] Replace jQuery.text function with native function
Currently we are using jQuery to update DOM contents but native browser functions exist to do this in a standard way so we don't need to rely on jQuery for this.

Example: 
https://github.com/Techtonica/keyboard-shortcuts-practice/blob/1a642f3ce4a136ec3b325beec781f4ab97b18935/public/scripts/main.js#L213

Search for all usages of the `.text(` function and replace with standard functionality.

[StackOverflow code example](https://stackoverflow.com/questions/1358810/how-do-i-change-the-text-of-a-span-element-using-javascript)

### PR DESCRIPTION
**Pull request by dapper-gh**
Sorry @linhhvo! I didn't even notice that you were assigned until I had already pretty much finished the PR. Anyways, this fixes #104. It doesn't just do .text functions but mostly eliminates any trivial use of jQuery. I also spun up a quick development server and it seemed to work. This fix recreates some functions (such as toggleClass), and uses some semi-new tech (such as document.querySelector).

### CODE REVIEW

**Review by alodahl — COMMENTED** [review]
Is there a reason I still see things like `  let height = $(window).height();` in main.js?  Is that file more complicated to refactor?

**Comment by dapper-gh** [comment]
Yeah, some functions are a little bit harder to refactor. I can change them if needed, but I was mostly focusing on DOM manipulations (like setting text, adding/removing classes, etc.)

File: public/scripts/main.js [hunk 1]
```diff
@@ -377,4 +418,4 @@ window.addEventListener('focus', function (e) {
   }
 });
 
-sequelize.close(); 
+sequelize.close();
```
Comment (alodahl) [inline public/scripts/main.js:421]: Could you delete this line?  I believe its a mistake anyway.

**Review by alodahl — APPROVED** [review]
Thanks for the help!  We'll merge after a second approval.

**Comment by alodahl** [comment]
Can you also resolve the conflicts please @dapper-gh ?

**Review by tilaprimera — APPROVED** [review]
Great @dapper-gh  Looks good!

**Review by Shraddha2104 — APPROVED** [review]
Looks great!

**Pull request merged**